app/views/pages/_locale_dropdown.html.erb

```diff
@@ -0,0 +1,9 @@
+  <div class="select language--select">
+    <select class="select__element toolbar__item text--small language--selector"
+            name="language_select"
+            id="language_select"
+            onchange="javascript:location.href = this.value;">
+      <option <%="selected disabled" if I18n.locale == :en %> value="<%=url_for(locale: :en) %>">English</option>
```

Comment:
I'm leaving this file in here so we can quickly switch back and because the stylesheet has the appropriate styling to make this look correctly